cmd/server/cadence/server.go

```diff
@@ -279,26 +277,21 @@ func (s *server) startService() common.Daemon {
 }
 
 func (*server) newMethod(
```

Comment:
I guess I did this typename, could we change it to `wraprings` or something like that?

Reply:
I want to leave it for now, since it is unrelated to my change. But I agree - the name is misleading.